packages/server/graphql/public/mutations/helpers/publishTeamPromptMentions.ts

```diff
@@ -0,0 +1,54 @@
+import {JSONContent} from '@tiptap/core'
+import getRethink from '../../../../database/rethinkDriver'
+import NotificationResponseMentioned from '../../../../database/types/NotificationResponseMentioned'
+import {TeamPromptResponse} from '../../../../postgres/queries/getTeamPromptResponsesByIds'
+
+const getMentionedUserIdsFromContent = (content: JSONContent): string[] => {
+  if (content.type === 'mention' && content.attrs?.id) {
+    return [content.attrs?.id]
+  }
+
+  if (!content.content) {
+    return []
+  }
+
+  return content.content.map(getMentionedUserIdsFromContent).flat()
+}
+
+const publishTeamPromptMentionNotifications = async (
```

Comment:
+1 this function isn't actually calling `publish`, maybe call it `insert`?

Reply:
Renamed to `create`